master_changes.md

```diff
@@ -71,6 +71,7 @@ users)
 ## External dependencies
 
 ## Format upgrade
+  * Fix switch and repo format upgrade on Windows. A block occurred because the global lock fd was reopened instead of using the one already opened.  [€6839 @rjbou]
```

Comment:
```suggestion
  * Fix switch and repo format upgrade on Windows. A block occurred because the global lock fd was reopened instead of using the one already opened.  [#6839 @rjbou]
```